config/maintainers.json

```diff
@@ -1,35 +1,35 @@
 {
```

Comment:
formatted by configlet fmt.
Same reasoning as above.

Reply:
It could be included in the new PR with configlet changes.

Reply:
dito